### ISSUE TITLE
Accessibility: Make Tab key focus restricted to the currently open view

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#181139]
CC @meganrogge and @isidorn

For blind users, overlaid views might be very confusing if they lack distinct focus boundaries.

I believe pressing the Tab/Shift+Tab keys within Ctrl+F should remain in this view and cycle through related controls, unless users explicitly close the view.

Ctrl+F view is just one of the examples.

## Current Behavior

Consecutive Tab/Shift+Tab presses fall to the editor area.

## Expected/Suggested Behavior

Tab key focus is restricted to the currently open view.

## Related Resource

https://stackoverflow.com/questions/50178419/how-can-restrict-the-tab-key-press-only-within-the-modal-popup-when-its-open

**Comment by meganrogge** [comment]
I believe we have historically wanted to make tabbing flow all the way through the app. As such, I am wondering if this would be disruptive to muscle memory. Would we do this in other components - like the panel - as well?

@isidorn thoughts?

**Comment by isidorn** [comment]
@meganrogge correct. The idea of tab navigation is to allow users to go all the way through the app.

I suggest that we leave this open as a feature request and see if we get similar feedback from other users.

**Comment by meganrogge** [comment]
@jooyoungseo now that we tried this with the terminal (with limited success), do you think we can close this? 

I believe that managing tab focus mode for each scope is a little confusing

**Comment by jooyoungseo** [comment]
@meganrogge This is what I discussed in our last meeting.
